src/tables/helpers/get-column-data.js

```diff
@@ -1,7 +1,7 @@
 import { castArray, has } from '../../utils'
 
 // Get column info from children via props
-function getColumnData(children = [], doDisable) {
+function getColumnData(children, doDisable) {
```

Comment:
This was only called in `sortable-table`, whose prop types **require** children. We were getting a flag for missing a branch here, but in reality this isn't a valid use of this util.